client/clip_client/client.py

```diff
@@ -1,7 +1,6 @@
 import mimetypes
```

Comment:
I think we can reuse some codes in this file since its getting quite big

Reply:
yes, exactly. But this is not the response of this PR

Reply:
OK but don't do it too soon 🥲pls wait for my PRs